src/KafkaFlow.Retry/Simple/RetrySimpleMiddleware.cs

```diff
@@ -51,9 +51,8 @@ public async Task Invoke(IMessageContext context, MiddlewareDelegate next)
                             }
                         }
 
-                        this.logHandler.Error(
+                        this.logHandler.Verbose(
```

Comment:
Hi ! We are also looking for this feature, but could we log the last attempt instead of the first one? If a message is retried only once and then passes, it means the issue is temporary and we don't have to deal with it, but if after all the retries we still can't consume, then we have a problem that we have to deal.